apps/src/templates/census/CensusForm.tsx

```diff
@@ -17,32 +16,78 @@ import {
 } from './censusQuestions';
 import SchoolAutocompleteDropdownWithLabel from './SchoolAutocompleteDropdownWithLabel';
 
-export const censusFormPrefillDataShape = PropTypes.shape({
-  userName: PropTypes.string,
-  userEmail: PropTypes.string,
-  isTeacher: PropTypes.bool,
-  schoolCountry: PropTypes.string,
-  schoolId: PropTypes.string,
-  schoolType: PropTypes.string,
-  schoolName: PropTypes.string,
-  schoolState: PropTypes.string,
-  schoolZip: PropTypes.string,
-});
-
-const CensusForm = ({
+const styles = formStyles as Record<string, CSSProperties>;
```

Comment:
fix for handling components with existing inline styles. unfortunately need to cast the type